get_last_revision_titles.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+"""
+    get_last_revision.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Revisions` module: Get data including content of last revision of titles [[API]] and [[Main Page]]
+    MIT license
+"""
+import requests
+
+req_ssn = requests.Session()
+
+api_url = 'https://www.mediawiki.org/w/api.php'
+titles = 'API|Main Page'
+rvprop = 'timestamp|user|comment|content'
+
+search_params = {
+    'action': 'query',
+    'prop': 'revisions',
+    'titles': titles,
+    'rvprop': rvprop,
+    'rvslots': 'main',
+    'format': 'json',
+    'formatversion': 2
+}
+
+data_req = req_ssn.get(url=api_url, params=search_params)
+data = data_req.json()
+
+title_data = data["query"]["pages"] # Gives a list containing the pages requested and their revisions
```

Comment:
For this particular example, we don't need intense data extraction for a particular page. We can just output the result and all the code after this very step can be removed.